Gemfile

```diff
@@ -21,5 +23,5 @@ group :test do
   gem 'minitest-reporters'
   gem 'rack-test'
   gem 'ruby_parser'
+  gem 'minitest-hooks'
```

Comment:
I'm not seeing changes in `Gemfile.lock`, are we not using a `lock` file?

Reply:
Oh man! so we ARE using a lock file, but it's not committed. I'm wondering if it should be a committed file?

Reply:
Yes! All the Ruby projects I have worked in use a `lock` file, helps ensure gem version is the absolute exact same across all environments and have a record in case of breaking change upgrades, so we know what version to rollback to if needed.

Reply:
@karloscarweber ^^ in case it gets lost

Reply:
I Added it!!!